src/api/arbo-typedefs.ts

```diff
@@ -10,6 +10,7 @@ export const arboTypedefs = `
     city: String
     state: String
     country: String!
+    countryAlphaTwoCode: String!
```

Comment:
Just helpful imo. I'd like to add the alpha 3 codes at some point too to make the map work a bit better.